### ISSUE TITLE
Misuse of circumference field due to positioning in mobile applications 

### ISSUE THREAD

**Issue by hadleystein1** [OpenTreeMap/otm-ios#321]
TreeFolks, an Austin, Texas-based customer, reported that users regularly put input the circumference into the diameter field, because the diameter field appears above circumference on the mobile application. 

Question from Tree Folks:
- Is it possible to make the diameter field editable like another field so that you can remove it from the app and/or drop and drag to the ideal location (i.e. below circumference)?

**Comment by dboyer** [comment]
I like the idea of letting the tree map owner customize whether diameter can be entered as diameter, circumference, or both. That's probably a feature that happens on the website in general and trickles down to the apps.

**Comment by hadleystein1** [comment]
I agree, @dboyer. I assume most trees are being added via mobile or bulk upload (something we will not know for sure until analytics are implemented), so it will perhaps be more important for the feature to trickle down to mobile than for it to be available on the Add A Tree process on the web application.
